libs/txnbuild/redeem_credit.go

```diff
@@ -42,6 +44,7 @@ func (redeemCredit *RedeemCredit) BuildXDR() (vxdr.VeloOp, error) {
 	return vxdr.VeloOp{Body: body}, nil
 }
 
+// FromXDR for RedeemCredit initialises the vtxnbuild struct from the corresponding xdr Operation.
```

Comment:
FromXDR for RedeemCredit initialises the vtxnbuild struct from the corresponding **XDR** Operation.